src/jqLite.js

```diff
@@ -169,20 +179,39 @@ var HTML_REGEXP = /<|&#?\w+;/;
 var TAG_NAME_REGEXP = /<([\w:-]+)/;
 var XHTML_TAG_REGEXP = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:-]+)[^>]*)\/>/gi;
 
+// Table parts need to be wrapped with `<table>` or they're
+// stripped to their contents when put in a div.
+// XHTML parsers do not magically insert elements in the
+// same way that tag soup parsers do, so we cannot shorten
+// this by omitting <tbody> or other required elements.
 var wrapMap = {
-  'option': [1, '<select multiple="multiple">', '</select>'],
-
-  'thead': [1, '<table>', '</table>'],
-  'col': [2, '<table><colgroup>', '</colgroup></table>'],
-  'tr': [2, '<table><tbody>', '</tbody></table>'],
-  'td': [3, '<table><tbody><tr>', '</tr></tbody></table>'],
-  '_default': [0, '', '']
+  thead: ['table'],
+  col: ['colgroup', 'table'],
+  tr: ['tbody', 'table'],
+  td: ['tr', 'tbody', 'table']
```

Comment:
Will using local variables improve compression?

```
var theadWrapper = ['table'];
var colWrapper = ['colgroup', 'table'];
var trWrapper = ['tbody', 'table'];
var tdWrapper = ['tr', 'tbody', 'table'];
var wrapMap = {
  thead: theadWrapper,
  colgroup: theadWrapper,
  caption: theadWrapper,
  tbody: theadWrapper,
  col: colWrapper,
  tr: trWrapper,
  td: tdWrapper,
  th: tdWrapper
};
```

Reply:
Why would that improve compression?

Reply:
Oh you are right. I thought that it would minify to something smaller. But in both cases the output looks basically like:

```
Da={
  thead:["table"],
  col:["colgroup","table"],
  tr:["tbody",
  "table"],
  td:["tr","tbody","table"]
};
Da.tbody=Da.tfoot=Da.colgroup=Da.caption=Da.thead;Da.th=Da.td;
```

Reply:
I expected that my way would come out as:

```
a=["table"],b=["colgroup","table"],c=["tbody","table"],d=["tr","tbody","table"],
Da={
  thead:a,
  tbody:b,
  tfoot:b,
  colgroup:b,
  caption:b,
  col:b,
  tr:c,
  td:d,
  th:d
};
```

Reply:
Where my version is actually about 4 characters shorter once you lose the whitespace :-)

Reply:
Although if each of the variable names `a`, `b`, `c` and `d` were more than one character in length the minifiers version is shorter.

Reply:
Ah, only now did I notice you provided the full object up front. But I guess your research indicates it's not worth it?

Reply:
Yep - scratch this idea :-)